tutorials/quickstart-node.md

```diff
@@ -0,0 +1,497 @@
+-------------+---------+------------+-------------------------------------------------------------------
+ plpgsql     | 1.0     | pg_catalog | PL/pgSQL procedural language
+ timescaledb | 1.7.1   | public     | Enables scalable inserts and complex queries for time-series data
+(2 rows)
+```
+
+#### Step 3: Create a table
+
+Now let's create a table and model called `page_loads` for our database 
+using the Sequelize command line tool:
+
+```bash
+npx sequelize model:generate --name page_loads --attributes userAgent:string,time:date
+```
+
+You should get a result similar to this:
+
+```bash
+Sequelize CLI [Node: 12.16.2, CLI: 5.5.1, ORM: 5.21.11]
+
+New model was created at [some path here] .
+New migration was created at [some path here] .
+```
+
+And finally, let's migrate our change and ensure that it is reflected in the
+database itself:
+
+```bash
+npx sequelize db:migrate
+```
+
+You should get a result that looks like this:
+
+```bash
+Sequelize CLI [Node: 12.16.2, CLI: 5.5.1, ORM: 5.21.11]
+
+Loaded configuration file "config/config.json".
+Using environment "development".
+== 20200528195725-create-page-loads: migrating =======
+== 20200528195725-create-page-loads: migrated (0.443s)
+```
+
+#### Step 4: Create a model for the table
+
+With the `node_test` database created and a `page_loads` table configured
+with a proper schema, we are ready to create the `PageLoads` model in our
+code. A model is an abstraction on the data stored in the table.
+
+Above our `app.use` statement, add the following to `index.js`:
+
+```javascript
+let PageLoads = sequelize.define('page_loads', {
+    userAgent: Sequelize.STRING,
+    time: Sequelize.DATE
+});
+```
+
+We will now be able to instantiate a `PageLoads` object and save it to the 
+database.
+
+### Generate hypertable [](create_hypertable)
+
+In TimescaleDB, the primary point of interaction with your data is a [hypertable][hypertable], 
+the abstraction of a single continuous table across all space and time 
+intervals, such that one can query it via standard SQL.
+
+Virtually all user interactions with TimescaleDB are with hypertables. Creating tables 
+and indexes, altering tables, inserting data, selecting data, etc. can (and should) 
+all be executed on the hypertable.
+
+A hypertable is defined by a standard schema with column names and types, with at 
+least one column specifying a time value.
+
+>:TIP: The TimescaleDB documentation on [schema management and indexing[indexing-api-guide] explains this in further detail. 
+
+Let's create this migration to modify the `page_loads` table and create a
+hypertable by first running the following command:
+
+```bash
+npx sequelize migration:generate --name add_hypertable
+```
+
+You should get a result that looks like this:
+
+```bash
+Sequelize CLI [Node: 12.16.2, CLI: 5.5.1, ORM: 5.21.11]
+
+migrations folder at [some_path] already exists.
+New migration was created at [some_path]/]20200601202912-add_hypertable.js .
+```
```

Comment:
Showing the output the first time is helpful but I don't think showing it after each time is necessary.